spec/rubocop/cop/lint/unneeded_disable_spec.rb

```diff
@@ -61,6 +61,17 @@
             end
           end
 
+          context 'itself' do
```

Comment:
> But if `Lint/UnneededDisable` is disabled and there are no other cops (unnecessarily) disabled, shouldn't the cop report itself? Of course, it can't. It's disabled. :smile:

That scenario played out in my head. This is why I questioned this fix slightly.

> Seriously though, you should add another cop on the same comment line to check that nothing is reported.

Good catch, I had to modify the code slightly to account for that scenario.